commons/types/account.go

```diff
@@ -276,6 +276,45 @@ func ToAccountsByName(name string, txn *badger.Txn) ([]*Account, error) {
 	return Accounts, nil
 }
 
+func AccountPaging(page int,txn *badger.Txn) ([]*Account, error){
+	var iteratorCount = 0
+	var firstItem int
+	if page == 0 {
+		return nil, ErrInvalidRequest
+	}else if page == 1{
+		firstItem = 1
+	} else{
+		firstItem = (page * 10) - 9
```

Comment:
The "10" here is page size? If so, we should make that a parameter and not just a number in a calculation; even if we don't expose a way to alter the value, it'll help clear up the code.